content/client-lib-development-guide/features.textile

```diff
@@ -225,6 +225,17 @@ h3(#rest-presence). Presence
 *** @(RSP4b3)@ @limit@ supports up to 1,000 items; if unspecified defaults to the REST API default (100)
 * @(RSP5)@ Presence Messages retrieved are decoded in the same way that messages are decoded
 
+h3(#rest-encryption). Encryption
+* @(RSE1)@ @Crypto::getDefaultParams@ function:
+** @(RSE1a)@ Returns a complete @CipherParams@ instance, using the default values for any field not supplied
+** @(RSE1b)@ Takes a hashmap (or language equivalent) consisting of any subset of @CipherParams@ fields that includes a @key@
+** @(RSE1c)@ The @key@ must be either a binary (e.g. a byte array, depending on the language), or a base64-encoded string. If the key is a string, the function should base64-decode it into a binary. Since the conversion to base64 is not under Ably control, this should be done liberally -- in particular, it should work with 'url-safe base64' (which uses @-@ and @_@ instead of @+@ and @/@) in addition to standard base64
```

Comment:
Please reference the RFCs for url-safe and "standard" base64.

I think "leniently" is more appropriate than "liberally".